coderd/rbac/policy/policy.go

```diff
@@ -209,7 +209,8 @@ var RBACPermissions = map[string]PermissionDefinition{
 		Actions: map[Action]ActionDefinition{
 			ActionAssign: actDef("ability to assign roles"),
 			ActionRead:   actDef("view what roles are assignable"),
-			ActionDelete: actDef("ability to delete roles"),
+			ActionDelete: actDef("ability to unassign roles"),
+			ActionCreate: actDef("ability to create/delete/edit custom roles"),
```

Comment:
Suggestion, probably out of scope: should we have a separate RBAC resource `UserRoleAssignment` instead?

Reply:
I think we can have 1 resource for site role assignment and create, and 1 resource for org assignment and create. But I am going to push this until I do the UX imo